code/modules/surgery/organs/cell_holder.dm

```diff
@@ -0,0 +1,136 @@
+/obj/item/organ/internal/cell_mount
+	name = "cell mount"
+	desc = "A small mechanical augment capable of holding a battery."
+	icon = 'icons/obj/robot_parts.dmi'
+	icon_state = "chest"
+	dead_icon = "chest"
+	organ_tag = "cell_mount"
+	parent_organ = "chest"
+	slot = "cell_mount"
+	status = ORGAN_ROBOT
+	var/obj/item/stock_parts/cell/cell
+
+/obj/item/organ/internal/cell_mount/full/New()
+	cell = new(src)
+	..()
+
+/obj/item/organ/internal/cell_mount/Destroy()
+	QDEL_NULL(cell)
+	return ..()
+
+/obj/item/organ/internal/cell_mount/attackby(obj/item/I, mob/user, params)
+	if(isscrewdriver(I) && cell)
+		to_chat(user, "<span class='notice'>You remove [cell] from [src].</span>")
+		cell.forceMove(get_turf(src))
+		cell = null
+		return
+	if(istype(I, /obj/item/stock_parts/cell))
+		if(!cell)
+			if(user.drop_item())
+				I.forceMove(src)
+				cell = I
+				to_chat(user, "<span class='notice'>You install [I] into [src].</span>")
+		else
+			to_chat(user, "<span class='notice'>[src] already has a cell.</span>")
+		return
+	return ..()
+
+/obj/item/organ/internal/cell_mount/on_life()
+	if(owner)
+		if(cell)
+			cell.use(1) // ~33 minutes to go from full to nothing
+			var/cellcharge = cell.charge/cell.maxcharge
+			switch(cellcharge)
+				if(0.75 to INFINITY)
+					owner.clear_alert("cell_organ")
+				if(0.5 to 0.75)
+					owner.throw_alert("cell_organ", /obj/screen/alert/lowcell, 1)
+				if(0.25 to 0.5)
+					owner.throw_alert("cell_organ", /obj/screen/alert/lowcell, 2)
+				if(0.01 to 0.25)
+					owner.throw_alert("cell_organ", /obj/screen/alert/lowcell, 3)
```

Comment:
Low charge should be better telegraphed - since it's lethal when it hits 0, with no other symptoms beforehand, I can foresee it sneaking up on players who're occupied with the rest of the round.